html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Coding Form</title>
+</head>
+
+<body>
+  <form action="" method="post">
+    <h1>Coding Form!</h1>
+    <label for="email">Email</label>
+    <br>
+    <input type="text" id="email" name="email" size="20">
+    <br>
+    <label for="coding"> Why is coding cool? </label>
+    <br>
+    <select id="coding" name="coding">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <br>
+    <br>
+    <label for="language">Favorite Language (choose one)</label>
+    <br>
+    <input id="html" type="radio" name="language" value="html" />
+    <label for="html">HTML</label>
+    <input id="css" type="radio" name="language" value="css" />
+    <label for="css">CSS</label>
+    <input id="javascript" type="radio" name="language" value="javascript" />
+    <label for="javascript">JavaScript</label>
+    <br>
+    <br>
+    <label for="information"> Would you like to sign up for more information?
```

Comment:
This element isn't actually labeling a single form control element, so it can just be a `p` element.